sale_require_ref/__manifest__.py

```diff
@@ -37,7 +37,7 @@
     'demo': [
         'demo/demo.xml',
```

Comment:
you split and change the deko file name but does not update it in the manifest. this is not let it to install the module. please fix